docs/guides/admin/docs/upgrade.md

```diff
@@ -48,6 +48,21 @@ Please make sure to compare your configuration against the current configuration
   [this document](https://github.com/elan-ev/opencast-studio/blob/2020-09-14/CONFIGURATION.md).
 
 
+### Wowza streaming configuration changes
+
+In [this pull request](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
+tenant-specific. Before, you had to specify `org.opencastproject.streaming.url` in `custom.properties`. Starting with
```

Comment:
Inline code blocks starting with `etc/` get special highlighting to explain possible locations for these files.

```suggestion
tenant-specific. Before, you had to specify `etc/org.opencastproject.streaming.url` in `custom.properties`. Starting with
```